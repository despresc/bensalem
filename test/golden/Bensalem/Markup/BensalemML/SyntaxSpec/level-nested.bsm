@#lvl1

@###lvl2

@##lvl3
