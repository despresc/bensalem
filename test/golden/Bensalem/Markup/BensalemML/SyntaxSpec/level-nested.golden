fromList
 [ ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 0
     , srcLine = 1
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 6
     , srcLine = 1
     , srcCol = 7
     }
    }
   , elementArgPresentation = PresentLevel
   , elementName = "lvl1"
   , elementAttrs = fromList []
   , elementArg = fromList
    [ ElementNode
     ( Element
      { elementTagPos = SrcSpan
       { srcSpanName = ""
       , srcSpanStart = SrcPos
        { srcOffset = 8
        , srcLine = 3
        , srcCol = 1
        }
       , srcSpanEnd = SrcPos
        { srcOffset = 16
        , srcLine = 3
        , srcCol = 9
        }
       }
      , elementArgPresentation = PresentLevel
      , elementName = "lvl2"
      , elementAttrs = fromList []
      , elementArg = fromList []
      }
     )
    , ElementNode
     ( Element
      { elementTagPos = SrcSpan
       { srcSpanName = ""
       , srcSpanStart = SrcPos
        { srcOffset = 18
        , srcLine = 5
        , srcCol = 1
        }
       , srcSpanEnd = SrcPos
        { srcOffset = 25
        , srcLine = 5
        , srcCol = 8
        }
       }
      , elementArgPresentation = PresentLevel
      , elementName = "lvl3"
      , elementAttrs = fromList []
      , elementArg = fromList []
      }
     )
    ]
   }
  )
 , LineEnd
 ]